doc/radosgw/placement.rst

```diff
@@ -149,6 +149,15 @@ Then provide the zone placement info for that storage class:
         --data-pool default.rgw.glacier.data \
         --compression lz4
 
+
+.. note::
+
+   **Adding Additional Object Data Pools**
+
+   RGW supports adding a storage class as a way to add additional object data pools. 
```

Comment:
Suggest: `RGW supports defining a non-default storage class as a way to utilize more than one object (bucket) data pool, potentially on different media or with different replication strategies.`